frontend/app/controllers/OffersAndCompetitions.scala

```diff
@@ -6,10 +6,12 @@ import model.ContentItemOffer
 
 trait OffersAndCompetitions extends Controller {
   def list = CachedAction { implicit request =>
+
     val results = GuardianContentService.offersAndCompetitionsContent.map(ContentItemOffer)
-      .filter { item =>
-        item.content.fields.map(_("membershipAccess")).isEmpty
-      }.filter(_.imgOpt.nonEmpty)
+      .filter(_.content.fields.map(_("membershipAccess")).isEmpty)
+      .filter(!_.content.webTitle.startsWith("EXPIRED"))
```

Comment:
This is the current approach (events already have EXPIRED in the title) but I don't love it. Would be ideal if there's something from CAPI that can appropriately flag that this is expired/out of date.